static/app/views/performance/onboarding.tsx

```diff
@@ -740,22 +754,17 @@ export function Onboarding({organization, project}: OnboardingProps) {
               {received ? (
                 <Button
                   priority="primary"
+                  busy={!traceId}
+                  title={traceId ? undefined : t('Processing trace\u2026')}
                   onClick={() => {
-                    navigate(
-                      {
-                        pathname: location.pathname,
-                        query: {
-                          ...location.query,
-                          guidedStep: undefined,
-                          table: Tab.TRACE,
-                        },
-                      },
-                      {replace: true}
-                    );
-                    window.location.reload();
+                    const params = new URLSearchParams(window.location.search);
+                    params.set('table', Tab.TRACE);
+                    params.set('query', `trace:${traceId}`);
+                    params.delete('guidedStep');
+                    window.location.href = `${window.location.pathname}?${params.toString()}`;
                   }}
                 >
-                  {t('Take me to traces')}
+                  {t('Take me to my trace')}
```

Comment:
since we are now setting trace to the query, this can be named 'take me to my trace', similar to 'take me to my profiling'